src/doc/book/crates-and-modules.md

```diff
@@ -190,13 +190,19 @@ mod farewells;
 ```
 
 Again, these declarations tell Rust to look for either
-`src/english/greetings.rs` and `src/japanese/greetings.rs` or
-`src/english/farewells/mod.rs` and `src/japanese/farewells/mod.rs`. Because
-these sub-modules don’t have their own sub-modules, we’ve chosen to make them
-`src/english/greetings.rs` and `src/japanese/farewells.rs`. Whew!
-
-The contents of `src/english/greetings.rs` and `src/japanese/farewells.rs` are
-both empty at the moment. Let’s add some functions.
+`src/english/greetings.rs`, `src/english/farewells.rs`,
+`src/japanese/greetings.rs` and `src/japanese/farewells.rs` or
+`src/english/greetings/mod.rs`, `src/english/farewells/mod.rs`,
+`src/japanese/greetings/mod.rs` and
+`src/japanese/farewells/mod.rs`. Because these sub-modules don’t have
+their own sub-modules, we’ve chosen to make them
+`src/english/greetings.rs`, `src/english/farewells.rs`,
+`src/japanese/greetings.rs` and `src/japanese/farewells.rs`. Whew!
```

Comment:
The `english` and `japanese` modules each contain two sub-modules - `greetings` and `farewells`. These means that the compiler must look for a total of eight different module files - four that reside directly under the `english` and `japanese` directories - `src/{english,japanese}/{greetings,farewells}.rs` and four (`mod.rs` files) that reside under the `greetings` and `farewells` directories - `src/{english,japanese}/{greetings,farewells}/mod.rs`. The documentation listed only four of these - `src/{english,japanese}/greetings.rs` and `src/{english,japanese}/farewells/mod.rs`, which confused me. I thought it would be good if we listed out all the files that the compiler possibly looks for to prevent further confusion.